starter-project-backend-g31/src/models/rating.ts

```diff
@@ -0,0 +1,37 @@
+import mongoose from "mongoose";
+import Joi from "joi";
+
+const ratingSchema = new mongoose.Schema(
+    {
+        articleID: {
+            type: String,
+            required: true
+        },
+        userID: {
+            type: String,
+            required: true
+        },
+        rating: {
+            type: Number,
+            required: true
+        },
+        ratedAt: {
+            type: Date,
+            required: true,
+            default: Date.now
+        }
+
+    }
+);
+
+export const Rating = mongoose.model("Rating", ratingSchema);
+
+export const validate = function(rating: any){
```

Comment:
mongoose has validation options as you did in the fields, would be repetitive to have join validation.